### ISSUE TITLE
nuxt build --spa doesn't work when router.base is added to the config

### ISSUE THREAD

**Issue by jpoo90** [nuxt/nuxt.js#4057]
### Version

[2](https://github.com/nuxt.js/releases/tag/2)

### Reproduction link

[https://github.com/jpoo90/test-nuxt-spa](https://github.com/jpoo90/test-nuxt-spa)

### Steps to reproduce

Clone the repo.
npm i 
npm run broken:build

surge dist/ 

Loading the pages throws an error.

### What is expected ?

The scripts in the generated HTML files should point to the appropriate URL so they can be loaded.

### What is actually happening?

The script tags generated include the `router.base` path. 

The scripts are stored in `/dist/_nuxt/*.js` but the generated HTML files are trying to load them from `dist/router.base/_nuxt/*.js` 

This is an example of the genereted `dist` folder. You can see an example of the generated script tags.  

```html
dist/
├── 200.html
├── _nuxt
│   ├── mainBundle.js
│   ├── bundleOne.js
│   ├── bundleTwo.js
│   ├── moreBundles.js
├── index.html
├── one
│   └── index.html // <script type="text/javascript" src="/app/_nuxt/bundleOne.js">
└── some
    ├── three
    │   └── index.html
    └── two
        └── index.html
```

```js
// nuxt.config.js

module.exports = {
        ...
        router: {
          base: '/app/',
        },
}
```

When the browser tries to load this, a 404 is returned because the files are not found. 

### Additional comments?

This is not happening when the router.base is not modified. If you run `npm run build` the file's tree is generated as expected. Let me know if I can clarify/help with anything.

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c7912">#c7912</a>)</em></sub></div>

**Comment by husayt** [comment]
This has been resolved in #4019

**Comment by manniL** [comment]
@husayt Thank you :relaxed:
